From 1fbc6758a0b0fc9fd1121edaa0b86c3064166476 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megi@xff.cz>
Date: Fri, 24 Jul 2026 19:24:46 +0200
Subject: [PATCH] ARM: dts: sunxi-h3-h5: Add USB OTG DMA register region

Describe the USB-OTG internal DMA engine's registers so the musb glue
can use it, following the A64 pattern. Covers both H3 and H5.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi b/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
index ebc64458084c..0591c318b63f 100644
--- a/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
+++ b/arch/arm/boot/dts/allwinner/sunxi-h3-h5.dtsi
@@ -273,7 +273,8 @@ msgbox: mailbox@1c17000 {
 
 		usb_otg: usb@1c19000 {
 			compatible = "allwinner,sun8i-h3-musb";
-			reg = <0x01c19000 0x400>;
+			reg = <0x01c19000 0x400>, <0x01c19500 0x100>;
+			reg-names = "mc", "dma";
 			clocks = <&ccu CLK_BUS_OTG>;
 			resets = <&ccu RST_BUS_OTG>;
 			interrupts = <GIC_SPI 71 IRQ_TYPE_LEVEL_HIGH>;
-- 
2.43.0

